src/commands/protocols/get/receiver/v1.0.0/v1-0-0-handle-get-request-command.js

```diff
@@ -84,38 +88,59 @@ class HandleGetRequestCommand extends HandleProtocolMessageCommand {
         //     }
         // }
 
-        const promises = [this.tripleStoreService.getAssertion(ual)];
+        // const promises = [this.tripleStoreService.getAssertion(ual)];
+        this.operationIdService.emitChangeEvent(
+            OPERATION_ID_STATUS.GET.GET_REMOTE_GET_ASSERTION_START,
+            operationId,
+            blockchain,
+        );
+        const assertion = await this.tripleStoreService.getAssertion(ual);
+        this.operationIdService.emitChangeEvent(
+            OPERATION_ID_STATUS.GET.GET_REMOTE_GET_ASSERTION_END,
+            operationId,
+            blockchain,
+        );
 
-        if (includeMetadata) {
-            promises.push(this.tripleStoreService.getKnowledgeAssetMetadata(ual));
-        }
+        this.operationIdService.emitChangeEvent(
+            OPERATION_ID_STATUS.GET.GET_REMOTE_GET_KA_METADATA_START,
+            operationId,
+            blockchain,
+        );
+        const knowledgeAssetMetadata = includeMetadata
+            ? await this.tripleStoreService.getKnowledgeAssetMetadata(ual)
+            : undefined;
+        this.operationIdService.emitChangeEvent(
+            OPERATION_ID_STATUS.GET.GET_REMOTE_GET_KA_METADATA_END,
+            operationId,
+            blockchain,
+        );
 
-        const [assertion, knowledgeAssetMetadata] = await Promise.all(promises);
+        // if (includeMetadata) {
+        //     promises.push(this.tripleStoreService.getKnowledgeAssetMetadata(ual));
+        // }
+
+        // const [assertion, knowledgeAssetMetadata] = await Promise.all(promises);
 
         const responseData = {
             assertion,
             ...(includeMetadata && knowledgeAssetMetadata && { metadata: knowledgeAssetMetadata }),
         };
 
+        await this.operationIdService.updateOperationIdStatus(
+            operationId,
+            blockchain,
+            OPERATION_ID_STATUS.GET.GET_REMOTE_PREPARE_MESSAGE_END,
+        );
+
         if (assertion.length) {
             await this.operationService.markOperationAsCompleted(
                 operationId,
                 blockchain,
                 responseData,
-                [
-                    OPERATION_ID_STATUS.GET.GET_LOCAL_END,
```

Comment:
Why did you remove this event?

Reply:
Seems like there is no GET_LOCAL_START to match it

Reply:
We no longer have this block after merging v8/develop

Reply:
We should mark it here as completed if we found data in local get
@zsculac can you open PR with fix for this